examples/rllib_training.ipynb

```diff
@@ -252,18 +262,19 @@
     "    object_store_memory=2_000_000_000,  # 2 GB\n",
     ")\n",
     "\n",
-    "ppo = PPO(config)\n",
-    "\n",
-    "# Train the policy as you see fit\n",
-    "for _ in range(10):\n",
-    "    ppo.train()\n",
-    "    ppo.checkpoint()\n",
+    "from ray import tune\n",
     "\n",
-    "ray.shutdown()\n",
-    "```\n",
+    "# Run the training\n",
+    "tune.run(\n",
+    "    \"PPO\",\n",
+    "    config=config.to_dict(),\n",
+    "    stop={\"training_iteration\": 500},  # Adjust the number of iterations as needed\n",
+    "    checkpoint_freq=10,\n",
+    "    checkpoint_at_end=True\n",
+    ")\n",
     "\n",
-    "Training on a reasonably modern machine, we can achieve 5M steps over 32 processors in 6\n",
-    "to 18 hours, depending on specific environment configurations."
+    "# Shutdown Ray\n",
+    "ray.shutdown()"
```

Comment:
My logic here was to avoid having ray boot and run when generating docs, but it would've caught these issues so I'm in favor of moving this to a code block, and our ipynb caching system will avoid it running too often. Go ahead and run `make html` from the docs directory to generate a cached version of the new output and add that to the PR (it will change a file in `bsk_rl/docs/build/doctrees/nbsphinx/examples/rllib_training.ipynb`.